webpack/components/Content/Details/ContentDetails.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Nav, NavItem, TabPane, TabContent, TabContainer, Grid, Row, Col } from 'patternfly-react';
+import { PropTypes } from 'prop-types';
+import { LoadingState } from '../../../move_to_pf/LoadingState/index';
+
+const ContentDetails = (props) => {
+  const { contentDetails, schema } = props;
+  const { loading } = contentDetails;
+
+  const tabHeaders = () => {
+    const tabs = schema.map(node => (
+      <NavItem key={node.key} eventKey={node.key}>
+        <div>{node.tabHeader}</div>
```

Comment:
So this map here needed a key or else there was this every list item must have unique key error I was hitting. So instead of using index which I was unsure of regarding ordering here in the tabHeader vs the tabPane for the key, I thought it would be better and more controlled to provide a key in the schema.